Banno/jha-design
# Update Gulp and other dependencies to latest versions

Resolve most [Dependabot alerts](https://github.com/Banno/jha-design/security/dependabot) by updating dependencies to latest versions. Based on work in #36.

Most of the changes in the diff are in generated code (e.g. minor differences in minified SVGs). Actual substantive changes are in [`package.json`](https://github.com/Banno/jha-design/pull/45/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&show-viewed-files=true#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) and [`gulpfile.js`](https://github.com/Banno/jha-design/pull/45/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&show-viewed-files=true#diff-25789e3ba4c2adf4a68996260eb693a441b4a834c38b76167a120f0b51b969f7).

Closes [CONTENT-1449]

## How to test

 1. Run `yarn`
 2. Run `gulp` or `./node_modules/.bin/gulp` and verify that it completes successfully. You can ignore the many deprecation warnings for `Using / for division outside of calc()`.
 3. Run `gulp dev` or `./node_modules/.bin/gulp dev` and open <http://localhost:7777> and verify that everything looks correct. (Note that there are some visual issues in the upper right; these are also present in `master`.)

    <img width="499" alt="Screenshot 2023-09-13 at 4 51 30 PM" src="https://github.com/Banno/jha-design/assets/117983/09e2dd6b-c4bf-4e38-ad94-ac4e3f10799d">


[CONTENT-1449]: https://banno-jha.atlassian.net/browse/CONTENT-1449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
